pygeos/test/test_io.py

```diff
@@ -158,6 +158,28 @@ def test_to_wkt_exceptions():
         pygeos.to_wkt(point, output_dimension=4)
 
 
+def test_to_wkt_point_empty():
+    assert pygeos.to_wkt(empty_point) == "POINT EMPTY"
+
+
+def test_to_wkt_geometrycollection_with_point_empty():
+    collection = pygeos.geometrycollections([empty_point, point])
+    assert pygeos.to_wkt(collection) == "GEOMETRYCOLLECTION (POINT EMPTY, POINT (2 3))"
```

Comment:
It looks like this is sensitive to GEOS version (according to the Travis-CI results).

It looks like this is `GEOMETRYCOLLECTION Z (POINT EMPTY, POINT (2 3))` and `GEOMETRYCOLLECTION (POINT EMPTY, POINT (2 3))` depending on the version.  

Perhaps just testing 
```
assert pygeos.to_wkt(collection).endswith('(POINT EMPTY, POINT (2 3))')
```

is sufficient to capture the substantive part of this test, and sidesteps version-specific issues?